src/test/kotlin/com/civelix/tetris/TetrominoTest.kt

```diff
@@ -11,17 +11,121 @@ class TetrominoTest {
         assertEquals(4, Tetromino.I.width)
         assertEquals(1, Tetromino.I.height)
 
+        assertEquals(2, Tetromino.O.width)
+        assertEquals(2, Tetromino.O.height)
+
+        assertEquals(2, Tetromino.L.width)
+        assertEquals(3, Tetromino.L.height)
+
+        assertEquals(3, Tetromino.T.width)
+        assertEquals(2, Tetromino.T.height)
+
+        assertEquals(3, Tetromino.S.width)
+        assertEquals(2, Tetromino.S.height)
+
         assertEquals(2, Tetromino.J.width)
         assertEquals(3, Tetromino.J.height)
 
-        // TODO: finish with other tetrominos.
+        assertEquals(3, Tetromino.Z.width)
+        assertEquals(2, Tetromino.Z.height)
+    }
+
+    @Test
+    fun rotateRight_O() {
+        assertEquals(Tetromino.O, Tetromino.O.rotateRight())
+
+        assertEquals(2,Tetromino.O.rotateRight().width)
+    }
+    @Test
+    fun rotateRight_J() {
+        var rotatedPiece = Tetromino.J
+        for(i in 0 until 3) {
+            rotatedPiece = rotatedPiece.rotateRight()
+            assertNotEquals(Tetromino.J, rotatedPiece)
+        }
+        assertEquals(Tetromino.J, rotatedPiece.rotateRight())
+
+        assertEquals(3,Tetromino.J.rotateRight().width)
```

Comment:
Nit: space after , (same below). Did you run the code cleaner?

Reply:
I was able to run the reformat code function.